### ISSUE TITLE
still maintained?

### ISSUE THREAD

**Issue by IzzySoft** [peterhearty/WipeFiles#5]
Just wondering if this project is still maintained, as there were no commits for about 3 years. If not, may I suggest to archive it (via its Github settings)?

**Comment by peterhearty** [comment]
I'm still willing to correct any bugs that might be reported (that is, if I can remember how to do it). But nobody's reporting any bugs, probably because nobody is using it.

Is it standard practice to archive in these circumstances?

**Comment by IzzySoft** [comment]
Not necessarily. I'd archive a project I don't plan to work on in the "close future". You say you'd still fix bugs (and probably also accept PRs). In that case I'd rather add a line to the Readme (ugh, if there were any) saying something like

> This app is not actively developed at this time. But I'm fully willing to review/merge PRs and fix reported bugs.

So nobody wonders (like I did), and the status is clear. Just a suggestion – my question is answered :wink:

Thanks!
